src/convert/streams.ts

```diff
@@ -219,10 +219,11 @@ export class ZipWriter extends ComponentWriter {
     callback(err);
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await
   public async _final(callback: (err?: Error) => void): Promise<void> {
     let err: Error;
     try {
-      await this.zip.finalize();
```

Comment:
this has not changed in ~2 years.  Given the bugs we've had with it in the other 2 places, I'm going to change it to be safe.

Reply:
This should have a comment like the one in `metadataApiDeploy.ts` lest someone get confused again and change it back.

Reply:
well, based on the NUTs, I just changed it back

Reply:
yeah it'd be good to include it with the eslint comment so that it also explains why we need that exception.  I didn't dig on it but pretty weird it didn't require the eslint comment in the other file as well.

Reply:
but I am adding a comment about it